x-pack/plugins/data_visualizer/public/application/index_data_visualizer/index_data_visualizer.tsx

```diff
@@ -56,6 +86,54 @@ export const DataVisualizerUrlStateContextProvider: FC<
     null
   );
 
+  const [currentSessionId, setCurrentSessionId] = useState<string | undefined>(undefined);
+
+  useEffect(() => {
+    if (search.session) {
```

Comment:
Should I be seeing an icon here? Also can you control the tooltip? Would be better if it was `ML` or even `Machine Learning` to match the label in the side nav.

![image](https://user-images.githubusercontent.com/7405507/146388983-b419e73a-42d5-413c-9f75-f3f57ff2654e.png)

Reply:
Update for icon and tooltip in https://github.com/elastic/kibana/pull/121053/commits/94fed45e2b8c8abc31f1c6f96bda964f7f0937f7 looks good.